src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.h

```diff
@@ -270,8 +269,8 @@ class vtkPlusDataCollectionExport vtkPlusCapistranoVideoSource: public vtkPlusUs
   PlusTrackedFrame::FieldMapType CustomFields;
 
 private:
-  vtkPlusCapistranoVideoSource(const vtkPlusCapistranoVideoSource &); // Not implemented
-  void operator=(const vtkPlusCapistranoVideoSource &); // Not implemented
+  vtkPlusCapistranoVideoSource(const vtkPlusCapistranoVideoSource&);  // Not implemented
+  void operator=(const vtkPlusCapistranoVideoSource&);  // Not implemented
 };
 
-#endif
+#endif
```

Comment:
Perhaps add a newline at the end?

Reply:
https://stackoverflow.com/a/37684763/276168

Reply:
Aaaahhh... I will stop removing them!